### PR TITLE
Add unit tests for `makeTopicMgtAPICall` method

### PR DESCRIPTION
**Pull request by omindu**

### CODE REVIEW

File: components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtil.java [hunk 1]
```diff
@@ -228,14 +231,14 @@ public void cancelled() {
     /**
      * Invoke the WebSub Hub to register, unregister the topics.
      *
-     * @param topic            topic name.
-     * @param webSubHubBaseUrl WebSub Hub base URL.
-     * @param operation        whether to register, deregister the queue.
-     * @throws IOException                  on errors communicate with
-     * @throws WebSubAdapterServerException
+ * @param topic                         topic name.
```
Comment (wenodpathirana) [inline components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtil.java:234]: Formatting

File: components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterConstants.java [hunk 2]
```diff
@@ -58,9 +58,13 @@ public enum ErrorMessages {
         ERROR_INVALID_EVENT_URI("60004", "Invalid event URI input", "Event URI input cannot be null or empty."),
         ERROR_INVALID_EVENT_TOPIC("60005", "Invalid event topic input", "Event topic input cannot be null or empty."),
         ERROR_INVALID_EVENT_ORGANIZATION_NAME("60006", "Invalid organization name input",
-                "Event organization name input cannot be null or empty"),
-        ERROR_WEB_SUB_BASE_URL_NOT_CONFIGURED("007", "Base url for WebSub Hub is not configured.",
-                "Base URL for WebSub Hub is not configured."),
+                "Event organization name input cannot be null or empty."),
+        ERROR_INVALID_TOPIC("60007", "Invalid WebSub hub topic input", "WebSub hub Topic cannot be null " +
```
Comment (LakshiAthapaththu) [inline components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterConstants.java:62]:
```suggestion
        ERROR_INVALID_TOPIC("60007", "Invalid WebSub hub topic input", "WebSub hub topic cannot be null " +
```

File: components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtil.java [hunk 3]
```diff
@@ -228,14 +231,14 @@ public void cancelled() {
...
+     * @param webSubHubBaseUrl          WebSub Hub base URL.
+     * @param operation                 whether to register, deregister the queue.
+     * @throws IOException              on errors while communicating with WebSu hub.
```
Comment (LakshiAthapaththu) [inline components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtil.java:237]:
```suggestion
     * @throws IOException              on errors while communicating with WebSub hub.
```

File: components/org.wso2.identity.outbound.adapter.websubhub/src/test/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtilTest.java [hunk 4]
```diff
@@ -18,29 +18,77 @@
...
+    private static final String TEST_TOPIC = "TEST-TOPIC";
+    private static final String TEST_ORG_NAME = "test-org";
+    private static final String TEST_PROPERTY = "test-property";
+    private static final int TEST_ORG_ID = 999;
+    private static final String INVALID_RESPONSE = "INVALID_RESPONSE";
+
+    private MockedStatic<HttpClientBuilder> mockStaticHttpClientBuilder;
+
+    private enum ResponseStatus {
+        STATUS_NOT_200, NULL_ENTITY, NON_SUCCESS_OPERATION
```
Comment (LakshiAthapaththu) [inline components/org.wso2.identity.outbound.adapter.websubhub/src/test/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtilTest.java:76]:
```suggestion
        
        STATUS_NOT_200, NULL_ENTITY, NON_SUCCESS_OPERATION
```

**Review by wenodpathirana — APPROVED** [review]

**Pull request merged**